packages/blade/src/components/Avatar/Avatar.web.tsx

```diff
@@ -20,24 +21,38 @@ const getInitials = (name: string): string => {
   return names[0][0] + names[names.length - 1][0];
 };
 
-const _Avatar = ({
-  name,
-  color = 'neutral',
-  size = 'medium',
-  variant = 'circle',
-  icon,
-  href,
-  target,
-  rel,
-  // Image Props
-  src,
-  alt,
-  srcSet,
-  crossOrigin,
-  referrerPolicy,
-  testID,
-  ...styledProps
-}: AvatarProps): ReactElement => {
+const _Avatar: React.ForwardRefRenderFunction<BladeElementRef, AvatarProps> = (
```

Comment:
Does this mean that we have to add these handlers for every component that could be a dropdown trigger? Even for custom components that users may have?

Reply:
yup. So custom components are likely to have it already since they won't be restrictive but for our internal components we do need them.

We have already added them in Link, Button for Tooltip so just needed to add in Avatar